accounts/ethstore/src/import.rs

```diff
@@ -46,21 +46,22 @@ pub fn import_accounts(src: &dyn KeyDirectory, dst: &dyn KeyDirectory) -> Result
 		.map(|a| a.address)
 		.collect::<HashSet<_>>();
 
-	accounts.into_iter()
-		.filter(|a| !existing_accounts.contains(&a.address))
-		.map(|a| {
-			let address = a.address.clone();
-			dst.insert(a)?;
-			Ok(address)
-		}).collect()
+	let mut out = Vec::with_capacity(accounts.len());
+	for account in accounts {
+		let address = account.address;
+		if existing_accounts.contains(&address) {
```

Comment:
to have the same behavior as before

```suggestion
		if !existing_accounts.contains(&address) {
```